Bryant_Trinh/for_loop_basic1.py

```diff
@@ -0,0 +1,27 @@
+#1
+for i in range(151):
+	print(i)
+#2
+for i in range(5,1000001,5):
+	print(i)
+#3
+for i in range(1,101):
+	if i % 5==0:
+		print("Coding")
```

Comment:
Hope you looked at Fizz buzz. Your output should be 1,2,3,4,Coding, 6,7,8,9, Coding dojo (not print 5 if you are printing Coding).